tests/test_multikernelmanager.py

```diff
@@ -613,8 +640,9 @@ async def test_bad_kernelspec_pending(self):
         assert kernel_id in km.list_kernel_ids()
         await ensure_future(km.shutdown_kernel(kernel_id))
         assert kernel_id not in km.list_kernel_ids()
+        km.context.destroy(linger=0)
 
-    @gen_test
+    @gen_test(timeout=30)
```

Comment:
```suggestion
    @gen_test(timeout=TIMEOUT)
```
Maybe ? `TIMEOUT=90` at top of file